packages/legacy_run_server.py

```diff
@@ -134,64 +125,31 @@ def try_update_log_level_from_config():
 
 
 def update_log_level_from_config():
+    global debug_update_timestamp
```

Comment:
Meine Idee dafür die Method in `try_update_log_level_from_config` und `update_log_level_from_config` zu splitten lag darin die Logik vom Error-Handling zu trennen um die beiden Funktionen übersichtlicher zu machen. Mit der neuen Änderung wandert Logik in die Funktion die bisher nur Error-Handling macht.

Schöner fände ich daher das in die try-Funktion zu packen:
```python
global debug_update_timestamp
conf_modification_timestamp = openwb_conf_path.stat().st_mtime
if debug_update_timestamp == conf_modification_timestamp:
    # config file has not changed, no need to read the file
    return
# [..]
debug_update_timestamp = conf_modification_timestamp
```
(Noch schöner fände ich das ganze in eine weitere Funktion auszulagern, siehe auch #2006)